extensions/ql-vscode/src/extension.ts

```diff
@@ -310,7 +310,7 @@ async function activateWithInstalledDistribution(ctx: ExtensionContext, distribu
   ctx.subscriptions.push(commands.registerCommand('codeQL.quickQuery', async () => displayQuickQuery(ctx, cliServer, databaseUI)));
   ctx.subscriptions.push(commands.registerCommand('codeQL.restartQueryServer', async () => {
     await qs.restartQueryServer();
-    const response = await Window.showInformationMessage('CodeQL Query Server restarted', 'Ok', 'Show Log');
+    const response = await Window.showInformationMessage('CodeQL Query Server restarted', 'Show Log');
```

Comment:
```suggestion
    const response = await Window.showInformationMessage('CodeQL Query Server restarted.', 'Show Log');
```

Nit: Add period for consistency with other informational dialogs like the "no updates available" dialog.